openreview/conference/templates/expertiseBidWebfield.js

```diff
@@ -0,0 +1,289 @@
+// ------------------------------------
+// Add EXPERTISE SELECTION Interface
+// ------------------------------------
+
+var CONFERENCE_ID = '';
+var HEADER = {};
+var SHORT_PHRASE = '';
+var BLIND_SUBMISSION_ID = '';
+var SUBMISSION_ID = '';
+var EXPERTISE_BID_ID = '';
+var EXPERTISE_SUBMISSION_ID = 'OpenReview.net/Archive/-/Direct_Upload';
+
+var BUFFER = 1000 * 60 * 30;  // 30 minutes
+
+// Main is the entry point to the webfield code and runs everything
+function main() {
+  Webfield.ui.setup('#invitation-container', CONFERENCE_ID);  // required
+
+  Webfield.ui.header(HEADER.title, HEADER.instructions);
+
+  renderSubmissionButton(EXPERTISE_SUBMISSION_ID);
+
+  Webfield.ui.spinner('#notes', { inline: true });
+
+  load().then(renderContent).then(Webfield.ui.done);
+}
+
+// Perform all the required API calls
+function renderSubmissionButton(INVITATION_ID) {
+  Webfield.api.getSubmissionInvitation(INVITATION_ID, {deadlineBuffer: BUFFER})
+    .then(function(invitation) {
+      Webfield.ui.submissionButton(invitation, user, {
+        onNoteCreated: function() {
+          // Callback funtion to be run when a paper has successfully been submitted (required)
+          promptMessage('Your submission is complete.');
+          load().then(renderContent).then(function() {
+            $('.tabs-container a[href="#allPapers"]').click();
+          });
+        }
+      });
+    });
+}
+
+function load() {
+
+  var authoredNotesP = Webfield.getAll('/notes', {
+    'content.authorids': user.profile.id,
+    noDetails: true,
+    sort: 'cdate'
+  })
+  .then(function(notes){
+    return notes.filter(function(note){
+      return !(note.original);
+    });
+  });
+
+  var edgesMapP = Webfield.getAll('/edges', {
+    invitation: EXPERTISE_BID_ID
```

Comment:
I noticed the same in the bid webfield (I'm fixing it in my branch), could you add the parameter to the query ```tail = user.profile.id```? we don't have to merge the edge results later.

Reply:
Done.